scripts/install-nix-from-closure.sh

```diff
@@ -247,3 +247,42 @@ variables are set, either log in again, or type
 in your shell.
 EOF
 fi
+
+added=
+p=$HOME/.nix-profile/etc/profile.d/nix.fish
```

Comment:
I'm assuming `nix-profile.fish.in` is converted to `nix.fish` somehow, but that's unclear.